github/actions_workflow_runs.go

```diff
@@ -94,6 +94,13 @@ type WorkflowRunAttemptOptions struct {
 	ExcludePullRequests *bool `url:"exclude_pull_requests,omitempty"`
 }
 
+// PendingDeploymentsRequest specificies body parameters to PendingDeployments
+type PendingDeploymentsRequest struct {
+	EnvironmentIDs []int64 `json:"environment_ids"`
+	State          *string `json:"state"`
+	Comment        *string `json:"comment"`
```

Comment:
Since these two last fields are required, let's please change them to `string` (instead of `*string`).